src/bin/vip-backup-db.ts

```diff
@@ -7,8 +7,13 @@ import { makeCommandTracker } from '../lib/tracker';
 
 const examples = [
 	{
-		usage: 'vip backup db @mysite.develop',
-		description: 'Trigger a new backup for your database of the @mysite.develop environment',
+		usage:
+			'vip @example-app.develop backup db\n' +
+			'      Generating a new database backup...\n' +
+			'      ✓ Preparing for backup generation\n' +
+			'      ✓ Generating backup\n' +
+			'      New database backup created',
+		description: 'Generate a new database backup for an environment.',
```

Comment:
This should probably match the "fresh database backup" language of `src/bin/vip-backup.ts`, right? Or vice versa.